src/renderers/common/XRRenderTarget.js

```diff
@@ -37,7 +37,7 @@ class XRRenderTarget extends RenderTarget {
 		 * @type {boolean}
 		 * @default false
 		 */
-		this.hasExternalTextures = false;
+		this._hasExternalTextures = false;
```

Comment:
Thanks for addressing the feedback although I had to be more precise^^. The properties in `XRRenderTarget` don't need the underscore since the entire class is meant to be internal. Only properties and methods added to the public classes like `Renderer` or `RenderTarget` should be private.

Reply:
Anyway, let's keep it like that for now. I can clean this up later as well.